cmd/serve/main.go

```diff
@@ -358,6 +360,10 @@ func (cmd *servecmd) ReadConfig() *config.AppConfig {
 		cmd.AppConfig.DNS.Upstream = strings.Split(cmd.AppConfig.DNS.Upstream[0], ",")
 	}
 
+    if len(cmd.AppConfig.ClientConfig.DnsServers) == 1 {
+        cmd.AppConfig.ClientConfig.DnsServers = strings.Split(cmd.AppConfig.ClientConfig.DnsServers[0], ",")
```

Comment:
maybe we should also use [TrimSpaces](https://pkg.go.dev/strings#TrimSpace) to cover `"1.1.1.1, 2.2.2.2"`

Reply:
On further review, I noticed its done the same way for `cmd.AppConfig.DNS.Upstream` and `cmd.AppConfig.VPN.AllowedIPs`. What do you think @nwaldispuehl ? Should we fix them as well?

Reply:
Good idea, will fix them all.